jdaviz/configs/default/plugins/viewers.py

```diff
@@ -185,6 +186,17 @@ def _on_subset_create(self, msg):
         if msg.subset.label not in self._expected_subset_layers and msg.subset.label:
             self._expected_subset_layers.append(msg.subset.label)
 
+    @property
+    def active_image_layer(self):
```

Comment:
This really only applies to Imviz, is it correct to have it so high up in `JdavizViewerMixin`?

Reply:
It's applicable to any image viewer.  This will just return None for a viewer without a valid image layer.  But would you prefer either putting this in the `AstrowidgetsImageViewerMixin` or creating a new mixin for non-astrowidgets image-viewer-specific methods/properties?